File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.py
n,m=input('').split()
arr=[]
for k in range(int(n)):
    arr.append(k+1)

for l in range(int(m)):
    i,j=input('').split()
    temp=arr[int(i)-1]
    arr[int(i)-1]=arr[int(j)-1]
    arr[int(j)-1]=temp

for k in range(int(n)):
    print(arr[k],end=' ')

File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int N, M;
    int i, j;

    int* arr;
    scanf("%d %d", &N, &M);
    arr = (int*)malloc(sizeof(int) * N);

    for (i = 0; i < N; ++i)
        arr[i] = i + 1;

    int temp;

    for (int I = 0; I < M; ++I)
    {
        scanf("%d %d", &i, &j);

        for (int K = 0; K < 1; ++K)
        {
            temp = arr[i-1];
            arr[i-1] = arr[j-1];
            arr[j-1] = temp;
        }
    }

    for (int i = 0; i < N; ++i)
        printf("%d ", arr[i]);

    free(arr);
    return 0;
}